front/cypress/e2e/content_builder/language_switch.cy.ts

```diff
@@ -0,0 +1,107 @@
+import { randomString } from '../../support/commands';
+
+describe('Content builder language switch', () => {
+  let projectId = '';
+  let projectSlug = '';
+  before(() => {
+    cy.setAdminLoginCookie();
```

Comment:
Is this needed for the `getAuthUser` call? I think we might be safe since it is in the `beforeEach`. Probably something to double check

Reply:
Just to update - I did end up needing this for the getAuthUser.